services/auth_service.py

```
@@ -0,0 +1,164 @@
+    new_user['_id'] = result.inserted_id
+    
+    return new_user
+
+
+def get_user_by_id(user_id):
+    """根据ID获取用户信息"""
+    try:
+        user = client.ht_server.users.find_one({"_id": ObjectId(user_id)})
+        if user:
+            user['_id'] = str(user['_id'])
+        return user
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)